.circleci/config.yml

```diff
@@ -79,19 +79,22 @@ installtorchcpuosx: &installtorchcpuosx
   run:
     name: Install torch CPU and dependencies
     command: |
-      pip3 install --progress-bar off https://download.pytorch.org/whl/cpu/torch-1.0.0-cp37-none-macosx_10_7_x86_64.whl
+      source /Users/distiller/miniconda/etc/profile.d/conda.sh
+      conda config --set always_yes yes
+      conda activate base
+      conda install --quiet 'pytorch>=1.1.0' -c pytorch
 
 installtorchcpu36: &installtorchcpu36
   run:
     name: Install torch CPU and dependencies
     command: |
-      pip3 install --progress-bar off https://download.pytorch.org/whl/cpu/torch-1.0.1.post2-cp36-cp36m-linux_x86_64.whl
+      pip3 install --progress-bar off 'torch>=1.1.0'
```

Comment:
Oh that's nice, they used to have separate CPU/CUDA distributions but I guess they unified?

Reply:
So for linux distributions they still have a separate specific command for a non-CUDA installation, but regardless if a GPU isn't found those tests are skipped and it is essentially run on the CPU. I used these commands since I felt they were easier to interpret and had no dammage of using them. If you would like me to change these back to the long urls with specific cpu only installation I can do that too!